ILRepack/Steps/SourceServerData/SourceFileDescriptor.cs

```diff
@@ -0,0 +1,34 @@
+using System.Diagnostics.Contracts;
+using System.Text.RegularExpressions;
+
+namespace ILRepacking.Steps.SourceServerData
+{
+    internal class SourceFileDescriptor
+    {
+        private static readonly Regex Regex = new Regex(@"([^\*]*)(\*([^\*]*))?");
```

Comment:
this is dropping the variables above var2 (srcsrv can have up 10 vars per file [cf](https://msdn.microsoft.com/en-us/library/windows/desktop/ms680641(v=vs.85).aspx))